pkg/kubefedctl/util/util.go

```diff
@@ -118,6 +121,22 @@ func HealthCheckRoleName(serviceAccountName, namespace string) string {
 	return fmt.Sprintf("kubefed-controller-manager:%s:healthcheck-%s", namespace, serviceAccountName)
 }
 
+// Checks if a resource with the given Kind and group is a Federated
```

Comment:
(No action required) Where possible, please try to avoid making unrelated non-functional changes in a given PR.

Also, please follow the golang standard for a function doc (e.g. `<method name> <does this>`).